fluent-syntax/test/fixtures_structure/expressions_call_args.json

```diff
@@ -0,0 +1,120 @@
+{
+  "body": [
+    {
+      "annotations": [],
+      "attributes": [],
+      "comment": null,
+      "id": {
+        "name": "key",
+        "span": {
+          "end": 3,
+          "start": 0,
+          "type": "Span"
+        },
+        "type": "Identifier"
+      },
+      "span": {
+        "end": 43,
+        "start": 0,
+        "type": "Span"
+      },
+      "type": "Message",
+      "value": {
+        "elements": [
+          {
+            "expression": {
+              "callee": {
+                "name": "FOO",
+                "span": {
+                  "end": 11,
+                  "start": 8,
+                  "type": "Span"
+                },
+                "type": "Function"
+              },
+              "named": [
+                {
+                  "name": {
+                    "name": "arg1",
+                    "span": {
+                      "end": 16,
+                      "start": 12,
+                      "type": "Span"
+                    },
+                    "type": "Identifier"
+                  },
+                  "span": {
+                    "end": 19,
+                    "start": 12,
+                    "type": "Span"
+                  },
+                  "type": "NamedArgument",
+                  "val": {
```

Comment:
If you rebase this PR on top of `feature/syntax-0.6` and re-generate the fxitures, `val` will become `value` and the test will pass. Travis tests the result of the merge rather than the actual PR, which is why this fails right now.